dds_web/api/files.py

```diff
@@ -30,6 +30,36 @@
 from dds_web.api.schemas import file_schemas
 from dds_web.api.schemas import project_schemas
 
+
+def check_eligibility_for_upload(status):
+    """Check if a project status is eligible for upload/modification"""
+    if status != "In Progress":
+        raise DDSArgumentError("Project not in right status to upload/modify files.")
+    return True
+
+
+def check_eligibility_for_download(status, user_role):
+    """Check if a project status makes it eligible to download"""
+    if status == "Available":
+        return True
+    if (status == "In Progress") and (user_role in ["Unit Admin", "Unit Personnel"]):
+        return True
+
+    raise DDSArgumentError("Current Project status limits file download.")
```

Comment:
Better! Can we do 
```
if status == "Available" or (status == "In Progress" and user_role in ["Unit Admin", "Unit Personnel"]): 
    return True
```

Reply:
It's not a deal breaker though, just a personal preference!

Reply:
Done.

Reply:
Great, thanks!